public/jwpltx.js

```diff
@@ -98,27 +150,51 @@ window.jwpltx = window.jwpltx || {};
         } else {
           uri.q = 32;
         }
-        uri.ti = Math.round(uri.vd / uri.q);
+
+        uri.ti = 0;
         uri.pw = 0;
-        sendData('s');
 
+        // Initialize latest quantile to compare further quantiles with
+        lastQ = getLastVODQuantile(vp, vd, uri.q);
```

Comment:
vd = 0
```suggestion
        lastQ = 0;
```

Reply:
This one is a bit tricky. We use `getLastVODQuantile` because a customer can start watching the video from Continue Watching section and in this case the value can be different.